src/Widgets/FormStepContainerWidget/components/ReviewComponent.tsx

```diff
@@ -17,19 +32,19 @@ export const Review = ({ widget, reviewData, onHide }) => {
     <Modal show={show} onHide={handleClose} size="lg">
       {widget.get("showReviewHeader") && (
         <Modal.Header style={{ alignSelf: "center" }}>
-          <Modal.Title>{widget.get("reviewHeaderTitle")}</Modal.Title>
+          <Modal.Title>{widget.get("reviewHeaderTitle") as string}</Modal.Title>
         </Modal.Header>
       )}
       <Modal.Body className="form-review">
-        {reviewData.map((step, i) => (
+        {reviewContent.map((steps, i) => (
```

Comment:
`<ReviewItem key={"item-" + i} item={d} />`

Using an index as a key in react is generally considered poor practice and should only be a last resort when no other unique values are available. The preferred approach is to use a persistent, unique identifier, such as an ID. This comes from how React manages and renders elements.

Here you have more info:
https://legacy.reactjs.org/docs/lists-and-keys.html#keys

Reply:
I'm aware of that but in this case, there is no unique identifier available which could be used, also the content mapping won't change, it's always fixed so there will not be any interchange between indexes or remove of content at a certain index.